### DISCUSSION TITLE
Markdown quick reference

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#27] [General]
These were linked in our assignments this week. I found them helpful. @metasean Is this where you want us to leave notes of this nature?

https://github.com/Cheatsheet-lang/Git-markdown 
https://learnmarkdown.com/

**Comment by metasean** [comment]
@Skye404 - These are great resources to be added to the [class-repo](https://github.com/USU-ITLS-x265-23-summer/class-repo)!

You'll need to make a PR with those links added.  (I'd suggest you add them to the [References/Resources.md](https://github.com/USU-ITLS-x265-23-summer/class-repo/blob/main/References/Resources.md)

**Reply by Skye404** [reply]
I believe I just created a pull request to add them to that location, References/Resources.md. Can you let me know if I did that correctly?
